core/src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -261,9 +262,12 @@ public void runInContext() {
           GrpcUtil.getClientStreamTracers(callOptions, headers, 0, false);
       String deadlineName =
           isFirstMin(callOptions.getDeadline(), context.getDeadline()) ? "CallOptions" : "Context";
+      Long nameResolutionDelay = callOptions.getOption(NAME_RESOLUTION_DELAYED);
       String description = String.format(
-          "ClientCall started after %s deadline was exceeded .9%f seconds ago", deadlineName,
-          effectiveDeadline.timeRemaining(TimeUnit.NANOSECONDS) / NANO_TO_SECS);
+          "ClientCall started after %s deadline was exceeded %.9f seconds ago. "
+              + "Name resolution took %.9f seconds.", deadlineName,
```

Comment:
I think this could be misunderstood. "Name resolution took .01 seconds", but name resolution could have taken 1s but this RPC started near the end of it. I'd suggest we emphasize this RPC's impact, like you do later with "Name resolution delay %.9f seconds."